plugins/UM3NetworkPrinting/PrinterInfoBlock.qml

```diff
@@ -290,7 +290,7 @@ Rectangle
 
                     Image
                     {
-                        width: 16 * screenScaleFactor
+                        width: statusText.height * screenScaleFactor
```

Comment:
Remove screenScaleFactor. screenScaleFactor is only used with absolute pixel values. statusText.height already has the devicePixelRatio included.

Reply:
Hey @fieldOfView hi :-)
Thanks I'll fix it.